examples/plot_multioutput_face_completion.py

```diff
@@ -39,10 +39,12 @@
 test = test[face_ids, :]
 
 n_pixels = data.shape[1]
-X_train = train[:, :np.ceil(0.5 * n_pixels)]  # Upper half of the faces
-y_train = train[:, np.floor(0.5 * n_pixels):]  # Lower half of the faces
-X_test = test[:, :np.ceil(0.5 * n_pixels)]
-y_test = test[:, np.floor(0.5 * n_pixels):]
+# Upper half of the faces
+X_train = train[:, :np.int64(np.ceil(0.5 * n_pixels))]
```

Comment:
We don't need `ceil` or I miss something ?
Just remove `np.int64(np.ceil(0.5 * n_pixels))` and replace it by `(n_pixels +1) // 2`.
```python
>>> np.int64(np.ceil(0.5 * 110))
55

>>> (110 + 1) // 2 
55

>>> np.int64(np.ceil(0.5 * 111))
56

>>> (111 + 1) // 2 
56
```